src/EditorFeatures/Test2/Rename/RenameTestHelpers.vb

```diff
@@ -74,7 +73,7 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Rename
             Assert.NotNull(result.LocalizedErrorMessage)
         End Sub
 
-        Public Async Function VerifyTagsAreCorrect(workspace As TestWorkspace, newIdentifierName As String) As Task
+        Public Async Function VerifyTagsAreCorrect(workspace As TestWorkspace, newIdentifierName As String, Optional expectFileRename As Boolean = False) As Task
```

Comment:
Can we make this a separate VerifyFileName operation? It really has nothing to do with tags, and it's a bit screwy that it's combined with this.